examples/rllab/figure_eight.py

```diff
@@ -1,6 +1,5 @@
-"""
-(description)
-"""
+"""Trains a subset of vehicles to increase traveling speeds in a figure 8."""
```

Comment:
"Trains a small percentage of autonomous vehicles" sounds less confusing than a subset